<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertDialogClass</class>
 <widget class="QDialog" name="ConvertDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>344</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ConvertDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="sourceFileLabel">
       <property name="text">
        <string>&amp;Source File:</string>
       </property>
       <property name="buddy">
        <cstring>sourceFilEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sourceFilEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="targetFormatLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Target Format:</string>
       </property>
       <property name="buddy">
        <cstring>targetFormatComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="targetFormatComboBox">
       <item>
        <property name="text">
         <string>BMP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EPS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GIF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JPEG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PNG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TIFF</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="enhanceCheckBox">
        <property name="text">
         <string>Enhance</string>
        </property>
        <property name="shortcut">
         <string>Alt+E</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="monochromeCheckBox">
        <property name="text">
         <string>Monochrome</string>
        </property>
        <property name="shortcut">
         <string>Alt+M</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTextEdit" name="outputTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>sourceFilEdit</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>targetFormatComboBox</tabstop>
  <tabstop>enhanceCheckBox</tabstop>
  <tabstop>monochromeCheckBox</tabstop>
  <tabstop>outputTextEdit</tabstop>
 </tabstops>
 <resources>
  <include location="ConvertDialog.qrc"/>
 </resources>
 <connections/>
</ui>
